interfaces/HO-Portal/src/app/user/user.page.ts

```diff
@@ -31,4 +32,29 @@ export class UserPage implements OnInit {
       }, 3000);
     });
   }
+
+  public checkConfirmPasswords() {
+    this.confirmPassword !== this.newPassword
+      ? this.newPasswordForm.form.setErrors({ differentPasswords: true })
+      : this.newPasswordForm.form.setErrors(null);
+  }
+
+  private shortPassword(): boolean {
+    return this.newPassword.length < 8;
```

Comment:
Here the same variable/constant should be used. Otherwise things will 'get out of sync' when only 1 of them is changed.

Reply:
Variable added